tests/lax_scipy_test.py

```diff
@@ -621,6 +621,34 @@ def test_vq(self, n_obs, n_codes, n_feats, dtype):
     self._CheckAgainstNumpy(osp_cluster.vq.vq, lsp_cluster.vq.vq, args_maker, check_dtypes=False)
     self._CompileAndCheck(lsp_cluster.vq.vq, args_maker)
 
+  def test_spence(self):
+    """Check against known values of Spence's function. SciPy implements
+    this test with the same values internally, but tests them against the
+    known analytic values. We test against SciPy values."""
```

Comment:
Rather than testing only against a 1D array of specific values, it would be better do do a parameterized test over values of different shapes and dtypes, just to make sure we're covering all corner cases.